steps/container_steps.py

```diff
@@ -22,11 +22,13 @@ def log_matches_regex(context, regex, timeout=TIMEOUT):
     if not run_log_matches_regex(context, regex, timeout):
         raise Exception("Regex '%s' did not match the logs" % regex)
 
+
```

Comment:
Whitespace changes are generally better off done as a seperate commit IMO.

Reply:
It is just PEP convention changes made by IDE when formatting it, I can just drop it but, IMHO, doing small formatting changes while fixing other stuff is helpful. your call.